src/api/app/models/token.rb

```diff
@@ -77,15 +77,17 @@ def owned_by?(some_user)
 #  string       :string(255)      indexed
 #  triggered_at :datetime
 #  type         :string(255)
+#  executor_id  :integer          default(0), not null, indexed
```

Comment:
Why don't you setup the association?

Reply:
If I setup the association now, then ActiveRecord is aware of it, and then every single spec detects the missing `executor` data (is a not null field, then it's required) and complains about it, then I have to adapt tons of specs (factories, controllers, etc) just to make ActiveRecord happy, because in the end I will replace `user` by `executor` in the specs, so those changes that I would need to do are not really necessary. Using just the _id field goes under the radar of ActiveRecord, allows me to populate the field and perform the backfilling. Nothing less, nothing more.

Reply:
That is why you see the weird magic above